paddle/fluid/framework/grad_op_desc_maker.h

```diff
@@ -38,10 +41,14 @@ class GradOpDescMakerBase {
   explicit GradOpDescMakerBase(
       const OpDesc& fwd_op, const std::unordered_set<std::string>& no_grad_set,
       std::unordered_map<std::string, std::string>* grad_to_var,
-      const std::vector<BlockDesc*>& grad_block = std::vector<BlockDesc*>())
+      const std::vector<BlockDesc*>& grad_block = std::vector<BlockDesc*>(),
+      const imperative::NameVarBaseMap* var_base_in = NULL,
```

Comment:
Remove `var_base_in` and `var_base_out`.

Reply:
done